src/i18n/lang/fr.json

```diff
@@ -259,7 +259,9 @@
   },
   "FileUploadCard": {
     "cancelMessage": "Annulé",
-    "errorMessage": "Le téléversement a échoué"
+    "errorMessage": "Le téléversement a échoué",
+    "of": "sur",
```

Comment:
I'm not sure "of" is the correct English word for this.
There are usually tools to do stuff like this in i18n.